src/parser/parseSVGDocument.ts

```diff
@@ -67,8 +68,9 @@ export async function parseSVGDocument(
   }
   const localClipPaths: Record<string, Element[]> = {};
   descendants
-    .filter((el) => el.nodeName.replace('svg:', '') === 'clipPath')
+    .filter((el) => getTagName(el) === 'clipPath')
     .forEach((el) => {
+      el.setAttribute('originalTransform', el.getAttribute('transform') || '');
```

Comment:
We need to save the original transform of this element because we may cycle over it multiple times and we need to then change its transform attribute.
We could do this ONCE instead of multiple times but that is a larger refactor not in scope for a fix.

Reply:
This is part 2 of the fix, even this you can still find it in 4.x @mikehollibaugh